src/Arcus.EventGrid.Tests.Integration/appsettings.json

```diff
@@ -1,12 +1,24 @@
 {
```

Comment:
So what I'm seeing is that you have:
- One Service Bus namespace with 2 topics
- One Event Grid topic with Event Grid schema
- One Event Grid topic with CloudEvents schema

If so, I'll add an issue to reflect this in our landscape

Reply:
I have two ServiceBus namespaces. I actually just deployed the test instracture twice but the second time with a CloudEvent input schema.

Reply:
Yeah we should consolidate those :)

Reply:
LGTM

Reply:
Ok, but definitly in another PR, since it's not even the final Azure resources.

Reply:
> Feel free to create an issue for the Azure changes and assign it to me

Done: https://github.com/arcus-azure/arcus.eventgrid/issues/127